### ISSUE TITLE
Make stored text downloadable/exportable

### ISSUE THREAD

**Issue by ntallman** [psu-libraries/cho#685]
Any text that is either extracted or OCR and then extracted by CHO and stored should be downloadable by users in the GUI and added as a new file in the appropriate fileset when exported to a bag. Text that is extracted (or OCR'ed and then extracted) by CHO is not stored as a file, but stored as a property, so these files need to be generated on the fly.

Done looks like:
  - text file appears as part of fileset on fileset show page, text file generated on the fly when download actuated by user through GUI
  - bag exports include a text file in the relevant fileset

Related: #645

**Comment by ntallman** [comment]
@cam156 was this implemented during MVP 4? I thought I remember @awead saying you switched course during the sprint and kept the file.

**Comment by ntallman** [comment]
#699 partially accomplishes this by creating the file and adding to the fileset. Other tickets should cover the other aspects. This ticket will likely be achieved when 1.x is ready.
